pzfx_parser.py

```diff
@@ -87,3 +98,13 @@ def read_pzfx(filename):
               for table in root.findall('Table')}
     
     return tables
+
+def convert_pzfx_to_excel(filename,output):
```

Comment:
Thanks for implementing this! However, I think it would be more useful if it could take a `tables` dictionary (i.e. the output of `read_pzfx` instead of a filename. This way, the user has an opportunity to make modifications to the tables before they are saved. This would allow this to be more of a utility function than an encapsulation. The user can then write `convert_pzfx_to_excel(read_pzfx(filename), output)`.

Instead of calling the output filename argument `output`, can you please rename it to `output_filename`? This makes it clear what type of input is needed. In python it is quite common for IO functions to take an open file descriptor instead, which makes `output` a bit ambiguous.

I.e. something like this:
```suggestion
def convert_pzfx_to_excel(tables, output_filename):
    """Takes a `tables` dict (from `read_pzfx`) and writes to an xlsx file, returns nothing."""
    with pd.ExcelWriter(output_filename, engine='openpyxl') as writer:
        for df_name, df in tables.items():
            for c in '\/*[]:?':
                df_name = df_name.replace(c, '')
            df.to_excel(writer, sheet_name=df_name)
    return
```